dali/util/cucontext.h

```diff
@@ -22,7 +22,7 @@ namespace dali {
 class CUContext {
  public:
   CUContext();
-  explicit CUContext(CUdevice device, unsigned int flags = 0);
+  explicit CUContext(int device_id_, unsigned int flags = 0);
```

Comment:
I don't like it. You're mixing runtime and driver APIs at CUContext API level. It's very likely to cause trouble unless CUdevice and device ordinal from CUDA runtime are the same (I don't think they are).

Reply:
Removing CUContext as it is no longer needed.